src/main.rs

```diff
@@ -176,8 +170,9 @@ fn run_shell(vm: &mut VirtualMachine) -> PyResult {
         println!("No previous history.");
     }
 
-    let ps1 = &objstr::get_value(&vm.sys_module.get_attr("ps1").unwrap());
-    let ps2 = &objstr::get_value(&vm.sys_module.get_attr("ps2").unwrap());
+    let sys_module = vm.sys_module.clone();
+    let ps1 = &objstr::get_value(&vm.get_attribute(sys_module.clone(), "ps1").unwrap());
+    let ps2 = &objstr::get_value(&vm.get_attribute(sys_module, "ps2").unwrap());
```

Comment:
CPython uses an empty string if somebody deletes `sys.ps{1,2}`, we could probably do the same.